kommunicate/build.gradle

```diff
@@ -8,17 +8,17 @@ rootProject.allprojects {
     }
 }
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     lintOptions {
         abortOnError false
     }
 
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 30
+        targetSdkVersion 31
         versionCode 1
-        versionName "2.2.5.1"
+        versionName "2.2.6"
```

Comment:
Should we change the version to 2.3.0 because of a major change(SDK version update)?

Reply:
Okay we should do that